schemas/schema-schema.ipldsch

```diff
@@ -365,42 +366,42 @@ type StructRepresentation union {
 
 ## StructRepresentation_Map describes a way to map a struct type onto a map
 ## representation.  For example, fields may be aliased,
-## or default values associated.
+## or implicit values associated.
 type StructRepresentation_Map struct {
 	fields optional {String:StructRepresentation_Map_FieldDetails}
 }
 ## StructRepresentation_Map_FieldDetails describes additional properties of a
 ## struct field when represented as a map.  For example, fields may be aliased,
-## or default values associated.
+## or implicit values associated.
 ##
-## If a default value is defined, then during marshalling, if the actual value
-## is the default value, that field will be omitted from the map; and during
-## unmarshalling, correspondingly, the absense of that field will be interpreted
-## as being the default value.
+## If an implicit value is defined, then during marshalling, if the actual value
+## is the implicit value, that field will be omitted from the map; and during
+## unmarshalling, correspondingly, the absence of that field will be interpreted
+## as being the implicit value.
 ##
-## Note that fields with defaults are distinct from fields which are optional!
+## Note that fields with implicits are distinct from fields which are optional!
 ## The cardinality of membership of an optional field is is incremented:
 ## e.g., the cardinality of "fieldname Bool" is 2; "fieldname optional Bool" is
 ## membership cardinality *3*, because it may also be undefined.
-## By contrast, the cardinality of membership of a field with a default value
+## By contrast, the cardinality of membership of a field with an implicit value
 ## remains unchanged; there is serial state which can map to an undefined value.
 ##
 ## Note that 'alias' supports exactly one string, and not a list: this is
```

Comment:
Not necessarily to tack onto this PR, but while I'm looking at it: @dignifiedquire suggested `rename` as an alternative name (ahheh) for `alias`.  ("Alias" is a bit problematic because for many readers it connotes that one of two names will be accepted.  "Rename" is what rust calls this.)

Reply:
I’m +1 on the change to rename, but want to note that this should go in ASAP so that I can use it before we finalize the unixfsv2 schema.

Reply:
it's in :thumbsup: